### DISCUSSION TITLE
clearing array errors

### DISCUSSION THREAD

**Discussion by Kiennyo** [react-hook-form/react-hook-form#2005] [Q&A] [answered]
Hello, I'm having trouble to delete error when there is an array, was wondering if it is possible to clearError like this:

`clearError('name[0]')`

**Comment by bluebill1049** [comment, marked as answer]
`clearError(['xxx', 'xxx'])`?

**Reply by Kiennyo** [reply]
Thanks for the reply this didn't worked out, when I'm printing it, the structure of the error is like this:

`{
  name[0]: 'Required',
  name[1]: 'Required'
}`

**Reply by bluebill1049** [reply]
do you have a codesandbox to reproduce?

**Reply by Kiennyo** [reply]
https://codesandbox.io/s/react-hook-form-controller-1krzt?file=/src/index.js

**Reply by bluebill1049** [reply]
https://codesandbox.io/s/react-hook-form-controller-9gbkv?file=/src/index.js
